### DISCUSSION TITLE
Welcome to learn Discussions!

### DISCUSSION THREAD

**Discussion by sapatevaibhav** [VaibhavCodeClub/learn#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sophiaodigie1** [comment]
hello I want to contribute to this project can you please assign me a task

**Reply by sapatevaibhav** [reply]
Yes surely go through all the issues and which one suits to your level comment there for assignment I will assign it to you.
